gdal/ogr/ogr_geometry.h

```diff
@@ -3253,7 +3253,7 @@ class CPL_DLL OGRGeometryFactory
     static OGRGeometry *createFromGML( const char * );
     static OGRGeometry *createFromGEOS( GEOSContextHandle_t hGEOSCtxt,
                                         GEOSGeom );
-    static OGRGeometry *createFromGeoJson( const char *);
+    static OGRGeometry *createFromGeoJson( const char *, size_t = -1 );
```

Comment:
Should rather be a int since that's what json_tokener_parse_ex() ultimately accepts